File: backend/src/Modules/Modules.Artwork.Presentation/Artworks/Create/CreateArtworkRequest.cs
namespace Modules.Artwork.Presentation.Artworks.Create;

public sealed record CreateArtworkRequest(
    string TitleHu,
    string? TitleEn,
    int? Year,
    string? TechniqueHu,
    string? TechniqueEn,
    decimal? WidthCm,
    decimal? HeightCm,
    string? DescriptionHu,
    string? DescriptionEn,
    bool IsFeatured,
    int DisplayOrder);

File: frontend/henn-laszlo/projects/artwork-data-access/src/lib/models/create-artwork-request.ts
export interface CreateArtworkRequest {
  readonly titleHu: string;
  readonly titleEn: string | null;
  readonly year: number | null;
  readonly techniqueHu: string | null;
  readonly techniqueEn: string | null;
  readonly widthCm: number | null;
  readonly heightCm: number | null;
  readonly descriptionHu: string | null;
  readonly descriptionEn: string | null;
  readonly isFeatured: boolean;
  readonly displayOrder: number;
}